numpy/lib/utils.py

```diff
@@ -690,7 +690,7 @@ def source(object, output=sys.stdout):
     Examples
     --------
     >>> np.source(np.interp)                        #doctest: +SKIP
-    In file: /usr/lib/python2.6/dist-packages/numpy/lib/function_base.py
+    In file: <__array_function__ internals>
```

Comment:
In colab
```python
>>> np.source(np.interp)
None
```
and from my local machine I get
```
<__array_function__ internals>
```

Reply:
This may be considered an issue with `np.source`, it could get the real file from `np.interp._implementation`. Another option is to deprecate the function, since it is just a thin wrapper around `inspect.getsource`. In lieu of actually fixing it, let's just leave this one out for now.

Reply:
reverted for now.